openc3-traefik/Dockerfile

```diff
@@ -17,9 +17,19 @@ EXPOSE 80
 # This default config needs to be removed for traefik to use our custom config instead
 USER root
 RUN rm /etc/traefik/traefik.toml || true
-USER ${USER_ID}:${GROUP_ID}
 
-# Update packages to eliminate CVEs if we're on docker.io (not ironbank)
+# Set user and group
+ENV IMAGE_USER=openc3
+ENV IMAGE_GROUP=openc3
+ENV USER_ID=1001
+ENV GROUP_ID=1001
+
+# Create non-root user and update packages if we're on docker.io (not ironbank)
```

Comment:
Because ironbank is already running non-root?

Reply:
Yes